src/tictactoe/Board.java

```diff
@@ -1,21 +1,116 @@
 package tictactoe;
 
+import java.util.*;
```

Comment:
instead of importing everything from `java.util.`, just import the things you're using from it.

Reply:
:+1:

Reply:
You should be able to edit the setting in IntelliJ to not do that.